addons/sms/tests/common.py

```diff
@@ -238,7 +250,11 @@ def assertSMSNotification(self, recipients_info, content, messages=None, check_s
 
         if messages is not None:
             for message in messages:
-                self.assertEqual(content, tools.html2plaintext(message.body).rstrip('\n'))
+                if from_text:
+                    with patch("odoo.tools.mail.tags_to_remove", ["html", "body", "a"]):
+                        self.assertEqual(content, tools.html2plaintext(tools.html_sanitize(message.body).rstrip('\n')))
+                else:
+                    self.assertEqual(content, tools.html2plaintext(tools.html_sanitize(message.body).rstrip('\n')))
```

Comment:
Have to say this kind of code leads to "I don't understand what the assert is doing" :D . Why do we support html input when odoo sends text (composer is text-based sms templates generate text, sms marketing generates text) ?

Reply:
Still this "from text" to try to cleanup, but probably once we agree on everything else